src/Standards/Squiz/Sniffs/Commenting/VariableCommentSniff.php

```diff
@@ -16,6 +16,13 @@
 class VariableCommentSniff extends AbstractVariableSniff
 {
 
+    /**
+     * Whether to use short forms of type keywords.
+     *
+     * @var boolean
+     */
+    public $useShortTypes = false;
+
```

Comment:
Remove this empty line.

```suggestion
```

Reply:
Please don't request changes which are in violation with the coding standard used by the library.